sygnal/__init__.py

```diff
@@ -16,7 +16,7 @@
 from importlib.metadata import PackageNotFoundError, version
 
 try:
-    __version__ = version(__name__)
+    __version__ = version("matrix-sygnal")
```

Comment:
Because it is `matrix-sygnal`, not `sygnal`?

Reply:
Precisely:

```
Python 3.11.6 (main, Oct  3 2023, 00:00:00) [GCC 13.2.1 20230728 (Red Hat 13.2.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sygnal
>>> sygnal.__name__
'sygnal'
>>> 

```